commit 185848707e73382ba930857502a0b80ff6039967
Author: Akinobu Mita <akinobu.mita@gmail.com>
Date:   Sun Jul 15 23:40:24 2007 -0700

    fault-injection: fix example scripts in documentation
    
    Fix and cleanup example scripts in fault injection documentation.
    
    1. Eliminate broken oops() shell function.
    
    2. Fold failcmd.sh and failmodule.sh into example scripts. It makes
       the example scripts work independent of current working directory.
    
    3. Set "space" parameter to 0 to start injecting errors immediately.
    
    4. Use /sys/module/<modulename>/sections/.data as upper bound of
       .text section. Because some module doesn't have .exit.text section.
    
    Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/Documentation/fault-injection/failcmd.sh b/Documentation/fault-injection/failcmd.sh
deleted file mode 100644
index 63177aba8106..000000000000
--- a/Documentation/fault-injection/failcmd.sh
+++ /dev/null
@@ -1,4 +0,0 @@
-#!/bin/bash
-
-echo 1 > /proc/self/make-it-fail
-exec $*